packages/react-core/src/components/FileUpload/examples/FileUploadCustomPreview.tsx

```diff
@@ -29,7 +29,7 @@ export const CustomPreviewFileUpload: React.FunctionComponent = () => {
     >
       {value && (
         <div className="pf-u-m-md">
-          <FileUploadIcon size="lg" /> Custom preview here for your {value.size}-byte file named {value.name}
+          <FileUploadIcon width="2em" height="2em" /> Custom preview here for your {value.size}-byte file named {value.name}
```

Comment:
I'm not sure why this change, but I'm also not a React dev :-)

Reply:
The `size` prop was removed in `createIcon.tsx`, so I had to set this custom styling here to match the styling we had before @srambach - not sure if there is a better way to do this

Reply:
Ok, I see that now. 🤷‍♀️